Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -562,6 +620,91 @@ def update_stats_model(self):
         self.error("\n".join(errors), shown=bool(errors))
         self.Warning.scores_not_computed(shown=has_missing_scores)
 
+    def update_comparison_table(self):
+        self.comparison_table.clearContents()
+        if self.resampling != OWTestLearners.KFold:
+            return
+
+        slots = self._successful_slots()
+        scores = self._scores_by_folds(slots)
+        self._fill_table(slots, scores)
+
+    def _successful_slots(self):
+        model = self.score_table.model
+        proxy = self.score_table.sorted_model
+
+        keys = (model.data(proxy.mapToSource(proxy.index(row, 0)), Qt.UserRole)
+                for row in range(proxy.rowCount()))
+        slots = [slot for slot in (self.learners[key] for key in keys)
+                 if slot.results is not None and slot.results.success]
+        return slots
+
+    def _scores_by_folds(self, slots):
+        scorer = self.scorers[self.comparison_criterion]()
```

Comment:
Can it?

Reply:
I just noticed `self.comparison_criterion` setting does not work.
But if it worked and if someone removes scores from scoring module, it probably can.

Reply:
It is now a schema-only setting, with pending. I added an out-of-range check. If scorers are added or removed, the setting can still be wrong (because it's an index) but it won't crash. I don't find this issue too important.